README.md

```diff
@@ -449,7 +422,51 @@ FIXME: we should clarify this section and add mention of Samsung Browser as it's
    browser to a backend endpoint that
    redirects the browser to a URL with the intent
    scheme it is possible to use an existing AS
-   without modifications - though compared to App Links redirecting via the web browser is a poorer UX and leaves a tab open in the user's browser.
+   without modifications.
+
+   **Note:** Since the Intent scheme is handled as
+   a normal custom scheme outside of Android, it is
+   important that the endpoint that redirects the
+   user tries to determine whether the request came 
+   from an Android device or another device.
```

Comment:
I think it's not just "android device", it's "android device *and* a browser that supports the intent url scheme"?

I guess there are 3 possible outcomes for Android browsers:

1) supports intent: urls and will launch the app
2) is aware of intent: urls, but they may be disabled by the user and hence not work
3) is unaware of intent: urls and broadcasts it as a custom scheme for any app that claims to handle intent: to pickup

Is that right? Or is the OS layer architected in such a way that '3' always results in the correct app launching? (Sorry, I just don't know)

Reply:
I am not exactly sure how the intent:// scheme is handled but I could not find a browser that does not support the intent scheme. The only issue I have experienced is that some browsers (Firefox) redirect the user to the fallback URL even if the app is installed.

Reply:
I had a quick check and things like https://www.mbsd.jp/Whitepaper/IntentScheme.pdf imply it is handled inside the browser, albeit it's widely supported. I think we should change 'android device' to "a browser running on Android that supports the intent: uri scheme". "Android browsers that don't support intent: Uris" is probably a very small attack surface though, what do you think?

Reply:
I think this is a good description. I have changed the text here.